ansible/setup.yml

```diff
@@ -140,12 +167,12 @@
       tags:
         - ansible-facts
 
-    - name: Increase ulimit
-      pam_limits:
-        domain: "{{ project_user }}"
-        limit_type: soft
-        limit_item: nofile
-        value: 24576
+#    - name: Increase ulimit
+#      pam_limits:
+#        domain: "{{ project_user }}"
+#        limit_type: soft
+#        limit_item: nofile
+#        value: 24576
```

Comment:
i think this ulimit is still "required" by node

Reply:
yeap, it was just ocal debugging 🗡️